migrations/20161007172819-lastEditedBy-in-Groups-migration.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.addColumn(
+      'Groups',
+      'lastEditedById',
```

Comment:
I think we should name this `lastEditedByUserId` to be very explicit and avoid confusion